wsgi/odyssy/people/models.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import unicode_literals
+from django.core.urlresolvers import reverse
+
+from django.db import models
+
+
+class People(models.Model):
+    class Meta:
+        ordering = ('name',)
+        verbose_name = 'People'
+        verbose_name_plural = 'People'
+
+    STATUS_CHOICES = (
+        ('faculty', 'Faculty'),
+        ('staff', 'Staff'),
+        ('visiting_faculty', 'Visiting Faculty'),
+    )
```

Comment:
Sir, there is a field for a specific post, in which you can add "Director", 'Registrar'.
And this 'STATUS_CHOICES' is only used to partitions all people in 3 views.